docs/standard/linq/add-elements-attributes-nodes-xml-tree.md

```diff
@@ -80,7 +80,7 @@ Dim xmlTree As XElement = _
 xmlTree.Add(<NewChild>new content</NewChild>)
 xmlTree.Add( _
     From el In srcTree.Elements() _
-    Where CInt(el) > 3 _
+    Where CInt(el) > 3
```

Comment:
Why was this one removed by not the one on lines 81 and 82?

Reply:
They were forgotten 😄 
Thanks for catching, will update.